components/prism-renpy.js

```diff
@@ -0,0 +1,28 @@
+Prism.languages.renpy= {
+
+	'comment': {
+		pattern: /(^|[^\\])#.*?(\r?\n|$)/,
+		lookbehind: true
+	},
+
+	'string': /"""[\s\S]+?"""|'''[\s\S]+?'''|("|')(\\?.)*?\1|(^#?(?:(?:[0-9a-fA-F]{2}){3}|(?:[0-9a-fA-F]){3})$)/,
```

Comment:
Did you really mean to use the `$` anchor without the `m` flag? This implies those hexadecimal patterns can only appear at the very end of the code.